core/amber/src/main/scala/edu/uci/ics/texera/web/TexeraCompilerWebApplication.scala

```diff
@@ -0,0 +1,85 @@
+package edu.uci.ics.texera.web
```

Comment:
instead of calling it WebApplication, we probably want to start naming those as `Service`?

Reply:
renamed